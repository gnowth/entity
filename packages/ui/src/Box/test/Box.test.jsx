import renderer from 'react-test-renderer';
import React from 'react';

import UIBox from '..';

it('renders correctly', () => {
  const tree = renderer.create(
    <UIBox />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
