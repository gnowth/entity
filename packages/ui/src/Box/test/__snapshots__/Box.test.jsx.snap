// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Box__UIBox-j97zbg-0 ldjLGq"
/>
`;
